sdk/eventgrid/ci.yml

```diff
@@ -21,21 +24,26 @@ pr:
     - release/*
   paths:
     include:
-    - sdk/eventgrid/
-    exclude:
-    - sdk/eventgrid/Azure.ResourceManager.EventGrid/
+    - sdk/eventgrid/ci.yml
+    - sdk/eventgrid/Azure.Messaging.EventGrid/
+    - sdk/eventgrid/Microsoft.Azure.EventGrid/
+    - sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/
+    - sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/
 
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
+    SdkType: alldata
```

Comment:
I'd like to scope this PR to simply splitting the mgmt and data. For now I'd suggest keeping the name client here and mgmtclient in the ci.mgmt.yml. While I agree it isn't 100% correct I'd like to avoid adding new SDKType for this one case.